src/scripts/install_machine.py

```diff
@@ -54,11 +54,21 @@ def get_graphics_capabilies(connection):
 
 
 def prepare_graphics_params(connection):
-    graphics_config = {
+    # Check to make sure loopback has an IPv6 address assigned
+    # if not default back to IPv4
+    ip_lo = subprocess.run(['ip', '-6', 'addr', 'show', 'dev', 'lo'], capture_output=True, text=True)
```

Comment:
Ah, haha: This doesn't yet work on Python 3.6:

   TypeError: __init__() got an unexpected keyword argument 'capture_output'"}"

I'll push a fix.

Reply:
Ohhhh gotcha, I'll keep that in mind for future and keep a better eye on when features were introduced. Sorry! Bare with me while I'm still getting to grips with it all

Reply:
Don't worry -- this happens to all of us *all the time*. That's why we have tests to tell us where we are wrong :grin: